errors.go

```diff
@@ -62,8 +43,8 @@ func newErr(errT errType, parents ...string) Err {
 	return *err
 }
 
-//Creates a new VARS error using the error given
-//If the error given was already a VARS error Prepend the parents and don't change the error
+// Creates a new VARS error using the error given
+// If the error given was already a VARS error Prepend the parents and don't change the error
 func newErrFromErr(err error, parents ...string) Err {
 	if varsErr, ok := err.(Err); ok {
```

Comment:
This is the check I was talking about that does the appending of parents for you. This method is called from the `appendFromErrs` and `appendFromError` function.